docs/user_docs/guides/abstraction.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Abstraction"
+sidebar_position: 3
+---
+
+## What is Abstraction
```

Comment:
The value of the good abstraction: 1. Separation of concerns, easy and friendly to understand based on identities/roles/scenarios 2. Shielding lower level details, preventing errors 3. Easier to use and automate, and has better portability (same as a good API). KCL cannot decide whether the abstraction made by the user is good or not, but it provides a technical means to do it.

Reply:
> The value of the good abstraction: 1. Separation of concerns, easy and friendly to understand based on identities/roles/scenarios 2. Shielding lower level details, preventing errors 3. Easier to use and automate, and has better portability (same as a good API). KCL cannot decide whether the abstraction made by the user is good or not, but it provides a technical means to do it.

fixed.